pages/thanks.tsx

```diff
@@ -21,7 +21,7 @@ export default function ThankYou(): JSX.Element {
           Your subscription is confirmed. You will start receiving our weekly issues now onwards.
         </Text>
         <div className="flex items-center justify-center">
-          <Link href="/issues">
+          <Link href="/issues" passHref>
```

Comment:
Its better if we wrap this button in anchor tag, `passHref` is only useful if the underlying element is an anchor tag
https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

Reply:
> 
> 
> Its better if we wrap this button in anchor tag, `passHref` is only useful if the underlying element is an anchor tag https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

Also if Link wraps a custom component as it seems, at least the rule seems to check for that.
https://nextjs.org/docs/messages/link-passhref

Reply:
Both explanations do not match as it seems.

Reply:
I think the `passHref` is only useful when the component is a styled component or a function component that eventually forwards the ref to an anchor tag like they [mentioned here](https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-function-component).

So you can just wrap the button in the anchor tag just like we did at rest of the places like here for example - 
https://github.com/scriptified/scriptified.dev/blob/b6e9b15ad7fd3dd442087f5b55fecde588580d6f/components/Footer.tsx#L45-L51